Global/JetBrains.gitignore

```diff
@@ -21,6 +21,14 @@
 .idea/**/gradle.xml
 .idea/**/libraries
 
+# Gradle and Maven with auto-import
+# When using Gradle or Maven with auto-import, you should exclude module files,
+# since they will be recreated, and may cause churn.  Uncomment if using
+# auto-import.
+# .idea/modules.xml
+# .idea/**/*.iml
+# .idea/libraries/**/*.xml
```

Comment:
I think this rule is covered by line 22

Reply:
I will ask JB, but this line might more precisely match what is mentioned in the original article.  Although, when I look at mine, the only files in that directory are XML files. :)